src/components/EkoSubtitles/EkoSubtitles.jsx

```diff
@@ -27,24 +27,30 @@ export function EkoSubtitles({style}) {
         throw new Error('This component needs to be wrapped in a player context, but one was not found');
     }
     let player = context && context.playerState && context.playerState.player;
+    let pluginInited = context && context.playerState && context.playerState.pluginInited;
+    
     useEffect(() => {
         if (!player) {
             return;
         }
-        const onSubtitlesInit = () => player.invoke('subtitles.mode', 'proxy');
+        if (pluginInited) {
+            pluginInited('subtitles').then((res) => {
+                player.invoke('subtitles.mode', 'proxy');
+                player.invoke('subtitles.visible', visible);
```

Comment:
Both `pluginInited` and `visible` should be added to the useEffect() dep array (2nd arg) at line 59.